src/lib/notify.js

```diff
@@ -140,10 +141,6 @@ class Notification extends BaseElement {
   }
 
   openSurvey(surveyUrl, sentiment, sitename) {
-    if (!storage.id) {
-      storage.id = {};
-    }
-    storage.id[this.id] = tabs.activeTab;
```

Comment:
I moved it down to `render`; this was the race condition this PR fixes. Sometimes the `activeTab` referenced in the former L146 ended up being the tab opened in the former L147.